prow/config.yaml

```diff
@@ -4970,6 +4970,31 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
 
   kubernetes-sigs/poseidon:
+  - name: pull-poseidon-bazel
+    agent: kubernetes
+    context: pull-poseidon-bazel
+    always_run: true
+    rerun_command: "/test pull-poseidon-bazel"
+    trigger: "(?m)^/test( all| pull-poseidon-bazel),?(\\s+|$)"
+    labels:
+      preset-service-account: true
+      preset-bazel-scratch-dir: true
+      preset-bazel-remote-cache-enabled: true
```

Comment:
Wanted to take a look at what needs caching first.
/hold cancel
This may not be helpful given the relatively low PR rate, it's fairly likely that cache results will go stale and be evicted. Caching also currently can hit bugs in Bazel that cause failures :/